doc/git_configuration.rst

```diff
@@ -2,3 +2,76 @@
 Git configuration
 =================
 
+In order to use git to contribute to ROSS project, follow the steps bellow:
+
+----------------------------------------
+Step 1: Make you own copy (fork) of ROSS
+----------------------------------------
+From the command line:
+
+::
+
+    git clone https://github.com/ross-rotordynamics/ross.git
```

Comment:
Here instead of 
`git clone https://github.com/ross-rotordynamics/ross.git`
It should be 
`git clone https://github.com/your-user-name/ross.git`
since the contributor will be cloning from the forked repository.